examples/streaming/kudu/src/main/scala/org/apache/gearpump/streaming/examples/kudu/Split.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.gearpump.streaming.examples.kudu
+
+import java.time.Instant
+
+import org.apache.gearpump.Message
+import org.apache.gearpump.streaming.source.DataSource
+import org.apache.gearpump.streaming.task.TaskContext
+
+class Split extends DataSource {
+
+  private var x = 0
+
+  override def open(context: TaskContext, startTime: Instant): Unit = {}
+
+  override def read(): Message = {
+
+    val tuple = ("column1" -> s"value$x", "column2" -> s"value2$x")
+    x+=1
```

Comment:
Will `x` ever overflow ?

Reply:
I just followed HBase example.

Reply:
Then it's an issue for HBase example too

Reply:
what is overflow??

Reply:
integer overflow

Reply:
you want me to put a stop to the integer
like if(x=100000) stop ??

Reply:
No, I think `x` needs to be a `Long`

Reply:
Done